app/controllers/api/v1/reports_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1
+    class ReportsController < ApplicationController
+      before_action :authorize_report!
+      after_action :verify_authorized
+
+      def index
+        case params[:report_on]
+        when 'movie'
+          @movies = Movie.all
+
```

Comment:
can you call a exists service for movie, because in 052/PR we are fixing n+1 problem, and when you reporting all data with n+1 and csv, this will kill the browser when there are more than 10000 requests

Reply:
and i think will be cool to add a show report for current movie or person, because I don't think that the editor will want to watch all 500 films in order to see the data of the film he needs

![image](https://user-images.githubusercontent.com/70888057/157561049-d78d9fd7-d88a-4ec5-8003-70fecf883b6e.png)